src/Core/Grid/Action/Row/AccessibilityChecker/DeleteCartAccessibilityChecker.php

```diff
@@ -33,11 +33,12 @@ final class DeleteCartAccessibilityChecker implements AccessibilityCheckerInterf
 {
     /**
      * {@inheritdoc}
+     *
+     * returns true if cart can be deleted or false if cart cannot be deleted
      */
     public function isGranted(array $record)
     {
         // if its number then the order has been placed so the cart cant be removed.
-        // this one is a little complicated so need a review to confirm it is correct
         return isset($record['status']) && !is_int($record['status']) && 0 == $record['status'];
```

Comment:
As I understand, if the `status` not exist, the cart can be deleted.
```php
return !isset($record['status']) || !is_int($record['status']) || 0 >= $record['status'];
```

Reply:
With the real data coming it works like: if the status is "Not placed" - meaning no order was placed on the cart - then it can be deleted; Otherwise no possibility to delete

Reply:
0 == $record['status'] with "12notready" gave false; All other tests passed

Reply:
@tdavidsonas88 I'm sorry I dont fully understand the topic 😄 is it solved/confirmed/answered ?